Formula/riemann.rb

```diff
@@ -5,28 +5,16 @@ class Riemann < Formula
   sha256 "fa2e22b712ed53144bf3319a418a3cd502ed00fa8e6bcb50443039a2664ee643"
   license "EPL-1.0"
 
-  bottle :unneeded
-
-  def shim_script
-    <<~EOS
-      #!/bin/bash
-      if [ -z "$1" ]
-      then
-        config="#{etc}/riemann.config"
-      else
-        config=$@
-      fi
-      exec "#{libexec}/bin/riemann" $config
-    EOS
-  end
+  depends_on "openjdk"
 
   def install
+    inreplace "bin/riemann", "$top/etc", etc
```

Comment:
Probably want to confirm this will behave similar to original shim script.

Example installed scripts.

- Env script:
    ```bash
    #!/bin/bash
    JAVA_HOME="${JAVA_HOME:-/usr/local/opt/openjdk/libexec/openjdk.jdk/Contents/Home}" exec "/usr/local/Cellar/riemann/0.3.6/libexec/bin/riemann"  "$@"
    ```
- First/Last few lines of inreplace'd script
    ```bash
    #!/usr/bin/env bash
    top="$(dirname "$0")/.."
    
    JAR="$top/lib/riemann.jar:$EXTRA_CLASSPATH"
    CONFIG="/usr/local/etc/riemann.config"
    COMMAND="start"
    AGGRESSIVE_OPTS="-server -XX:+UseConcMarkSweepGC -XX:+UseParNewGC -XX:+CMSParallelRemarkEnabled -XX:+AggressiveOpts -XX:+UseFastAccessorMethods -XX:+UseCompressedOops -XX:+CMSClassUnloadingEnabled"
    
    ...
    
    exec java $EXTRA_JAVA_OPTS $OPTS -cp "$JAR" riemann.bin "$COMMAND" "$CONFIG"
    ```